<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
border: 4px solid;
border-color: rgb(72, 57, 33);

background-color: qlineargradient(spread:pad, x1:0.482587, y1:1, x2:0.512, y2:0, stop:0 rgba(176, 128, 60, 255), stop:1 rgba(221, 162, 91, 255));

</string>
  </property>
  <widget class="QPushButton" name="add_sort_btn">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>414</y>
     <width>143</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton{border-radius: 20}</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string>Добавить сорт</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>526</width>
     <height>373</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 4px solid;
border-color: rgb(72, 57, 33);
border-radius: 20px;
background-color: rgb(184, 137, 71);
background-color: qlineargradient(spread:pad, x1:0.592, y1:1, x2:0.532, y2:0, stop:0.039801 rgba(214, 137, 68, 255), stop:0.512438 rgba(201, 134, 74, 251), stop:1 rgba(235, 179, 101, 255));</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>27</y>
      <width>271</width>
      <height>49</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;
background-color: rgb(255, 255, 255, 0);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhLatinOnly</set>
    </property>
    <property name="text">
     <string>Название сорта:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>3</number>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>63</y>
      <width>286</width>
      <height>76</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;
background-color: rgb(255, 255, 255, 0);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhLatinOnly</set>
    </property>
    <property name="text">
     <string>Степень обжарки(1, 2 или 3): </string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>3</number>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>135</y>
      <width>316</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;
background-color: rgb(255, 255, 255, 0);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhLatinOnly</set>
    </property>
    <property name="text">
     <string>Вид(молотый/зерновой):</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>3</number>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>180</y>
      <width>316</width>
      <height>79</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;
background-color: rgb(255, 255, 255, 0);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhLatinOnly</set>
    </property>
    <property name="text">
     <string>Описание вкуса:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>3</number>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>252</y>
      <width>136</width>
      <height>49</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;
background-color: rgb(255, 255, 255, 0);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhLatinOnly</set>
    </property>
    <property name="text">
     <string>Цена:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>3</number>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>297</y>
      <width>286</width>
      <height>82</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;
background-color: rgb(255, 255, 255, 0);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhLatinOnly</set>
    </property>
    <property name="text">
     <string>Объем упаковки:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>3</number>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="degree_of_roasting">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>96</y>
      <width>211</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid;
border-color: rgb(72, 57, 33);
border-radius: 4px;
background-color: rgb(255, 255, 255, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="shredded">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>147</y>
      <width>211</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid;
border-color: rgb(72, 57, 33);
border-radius: 4px;
background-color: rgb(255, 255, 255, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="taste">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>207</y>
      <width>211</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid;
border-color: rgb(72, 57, 33);
border-radius: 4px;
background-color: rgb(255, 255, 255, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="price">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>267</y>
      <width>211</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid;
border-color: rgb(72, 57, 33);
border-radius: 4px;
background-color: rgb(255, 255, 255, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="package_volume_grams">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>327</y>
      <width>211</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid;
border-color: rgb(72, 57, 33);
border-radius: 4px;
background-color: rgb(255, 255, 255, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>39</y>
      <width>211</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid;
border-color: rgb(72, 57, 33);
border-radius: 4px;
background-color: rgb(255, 255, 255, 0);</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>414</y>
     <width>143</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton{border-radius: 20}</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
